crates/process-event/src/main.rs

```diff
@@ -48,6 +53,11 @@ fn main() -> Result<(), anyhow::Error> {
         if dif_candidates {
             form = form.text("options", r#"{"dif_candidates":true}"#);
         }
+
+        if compare_stackwalking_methods {
+            form = form.text("options", r#"{"compare_stackwalking_methods":true}"#);
```

Comment:
This is now mutually exclusive with `dif_candidates`. Maybe we should actually jump through the hoop and have a proper JSON object for these.